XenAdmin/Program.cs

```diff
@@ -719,7 +719,8 @@ private static void FatalError()
             }
             else
             {
-                using (var dlg = new ErrorDialog(msg) {WindowTitle = Messages.MESSAGEBOX_PROGRAM_UNEXPECTED_TITLE})
+                using (var dlg = new ErrorDialog(msg)
+                    {WindowTitle = string.Format(Messages.MESSAGEBOX_PROGRAM_UNEXPECTED_TITLE, BrandManager.BrandConsole)})
                     dlg.ShowDialog();

```

Comment:
Ought the `dlg.ShowDialog();` be within the brackets of the `using` statement as the indentation suggests? Like this:
```csharp
                using (var dlg = new ErrorDialog(msg)
                {
                    WindowTitle = string.Format(Messages.MESSAGEBOX_PROGRAM_UNEXPECTED_TITLE, BrandManager.BrandConsole)})
                    dlg.ShowDialog();
                }
```
Otherwise it seems to suggest it may have been disposed of before it is shown.

This was the case before your modification however.

Reply:
Not within the same brackets as the WindowTitle, this does not compile. I suppose the line break after "(msg)" is a bit confusing to the eye, it's clearer without it:
```
using (var dlg = new ErrorDialog(msg){WindowTitle = string.Format(...)})
    dlg.ShowDialog();
```
or
```
using (var dlg = new ErrorDialog(msg){WindowTitle = string.Format(...)})
{
    dlg.ShowDialog();
}
```
which stands for:
```
using (var dlg = new ErrorDialog(msg))
{
    dlg.WindowTitle = string.Format(...);
    dlg.ShowDialog();
}
```

Reply:
Ah I see, I was confusing the curly brackets of the object setup (of `WindowTitle") for ones that scope the `using` statement.